### PR TITLE
Enable pip2 installation (Added setup.py)

### PR DESCRIPTION
**Pull request by Darkade**
This pull request includes a

- [x] New feature

The following changes were made

- Moved working code inside `pokemongomap/`
- Modified `.gitignore` to account for pip testing and egg file generation
- Added `setup.py` that includes an `entry_point` called `pokemap`
 - this allows you to use the map without `python2 runserver.py` and just use `pokemap`

This will make install easier for everyone, and allow us to distribute via Pypi

You can test this by running

```
pip2 install -e git+https://github.com/Darkade/PokemonGo-Map.git@setuppy#egg=pokemongo-map
```

And then running
```
pokemap -st <step count> -l <location> -u <ptc user> -p <ptc password>
```

You may need to manually add your `credentials.json`

### CODE REVIEW

**Comment by dancortes94** [comment]
Already tested it and works.

**Comment by MonkeyLeeT** [comment]
Does this one include all changes in #757 ?

**Comment by Darkade** [comment]
@MonkeyLeeT yes, it does

**Comment by MonkeyLeeT** [comment]
Please rebase. Since this is file structure change, need more testing here.

**Comment by Darkade** [comment]
@MonkeyLeeT rebased

**Comment by aescobedob** [comment]
Tested from develop a5d27f16a6d26cd9fd633d563d98315d0761709d, installs just fine. Great work. 

image

**Comment by Darkade** [comment]
Rebased

**Comment by AHAAAAAAA** [comment]
Putting this on hold till new develop release!

**Comment by JZ6** [comment]
needing a rebase

**Comment by Darkade** [comment]
Working on the latest rebase. Having some conflicts with the static files

**Comment by Darkade** [comment]
Rebase Succesful

**Comment by MonkeyLeeT** [comment]
Please rebase again.

**Comment by Darkade** [comment]
Rebased

**Comment by Darkade** [comment]
Rebased

**Comment by Darkade** [comment]
Rebased

File: pokemongomap/config/credentials.json.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"gmaps_key" : ""
+}
```
Comment (CorruptComputer) [inline pokemongomap/config/credentials.json.example]: This file was removed.

**Comment by andrewda** [comment]
Way too many file changes (most likely due to an incorrect rebase). I'm going to keep this open for a while, but if it's not fixed I'll go ahead and close it, because it might be easier to open a new PR.

**Pull request closed without merging**